content/api/commands/wait.md

```diff
@@ -122,17 +123,18 @@ cy.wait('@getBooks')
 // responded with an empty array first
 cy.get('#book-results').should('be.empty')
 
-// now re-define the /books response
-cy.intercept('/books', [{ name: 'Emperor of all maladies' }])
+// now the request (aliased again as `getBooks`) will return one book
```

Comment:
This comment and code line are causing horizontal scrolling at large viewport sizes. Can you tweak the text and book name just a little to help them fit?

Reply:
Done